### ISSUE TITLE
some assets in web Acceptance test didn't loaded because of bower not run in travis

### ISSUE THREAD

**Issue by samsonasik** [sitrunlab/LearnZF2#94]
it got error like this : 
```
[Sat Dec 27 13:33:53 2014] ::1:40989 [404]: /js/dependencies/prism/themes/prism-okaidia.css - No such file or directory
```
how about using **bowerphp** ? I tried it in local with jquery deps and it quite simple easy to setup : 
![bowerphp](https://cloud.githubusercontent.com/assets/459648/5601361/be4d081e-932c-11e4-8057-1740b7793486.png)

**Comment by samsonasik** [comment]
I give a try create PR #95 for it.

**Comment by samsonasik** [comment]
Fixed by #95
